test/sparse/CS/multiplication/correctness.chpl

```diff
@@ -0,0 +1,161 @@
+/* Matrix multiplication */
+
+use LayoutCS;
+
+config const correctness = false,
+             m = 6,
+             n = 7;
+
+proc main() {
+  const ADom = {1..m, 1..n},
+        BDom = {1..n, 1..m};
+
+  var csrDom: sparse subdomain(ADom) dmapped CS(),
+      cscDom: sparse subdomain(BDom) dmapped CS(row=false);
+
+  var csrArr: [csrDom] real,
+      cscArr: [cscDom] real;
+
+  var A: [ADom] real,
+      B: [BDom] real;
+
+  const rA1 = ADom.dim(1) by 2,
+        rA2 = ADom.dim(2) by 2;
+
+  csrDom += {rA1, rA2};
```

Comment:
Yep! `rA1` and `rA2` originally had different strides, but I settled on this for now.